design/oss_suite_release.md

```diff
@@ -0,0 +1,300 @@
+# Conjur OSS Suite Release
+
+## Feature Overview
```

Comment:
I like calling the first section "Motivation" or maybe "Outcome".  Providing some type of overview (as you do below) is needed, but I feel like it's often more important to answer the "why" about a project.